src/geo/leaflet/leaflet-layer-view-factory.js

```diff
@@ -5,12 +5,37 @@ var LeafletPlainLayerView = require('./leaflet-plain-layer-view');
 var LeafletCartoDBLayerGroupView = require('./leaflet-cartodb-layer-group-view');
 var LeafletTorqueLayerView = require('./leaflet-torque-layer-view');
 var LeafletCartoDBWebglLayerGroupView = require('./leaflet-cartodb-webgl-layer-group-view');
+// Should we expose a Tangram.cartodb method for doing this?
+var CCSS = require('tangram-cartocss');
 
-var LayerGroupViewConstructor = function (layerGroupModel, mapModel, options) {
-  if (options.vector) {
-    return new LeafletCartoDBWebglLayerGroupView(layerGroupModel, mapModel);
+// Make this configurable and crate a pipe of decision based on the configuration.
+var shouldLoadWithVector = function (metadata, cb) {
+  // NOTE: the limit should be setted by the server?
```

Comment:
So we are taking this decisions:

- Is mobile -> raster, others -> vector
- have less than 1M points -> vector, other -> raster
- Is cartocss supported -> vector, other -> raster

@javisantana